docs/sdks/sdk-ref-python.md

```diff
@@ -236,6 +238,30 @@ If you're using server-side integration (see [Publisher Integration Guide, Serve
         unmapped_identity = unmapped_identities.get("email1@example.com")
         reason = unmapped_identity.get_reason()
    ```
+### Monitor rotated salt buckets
+1. Create an instance of `IdentityMapClient` as an instance variable or reuse the one from [Map DII to raw UID2s](#map-dii-to-raw-uid2s) 
+   ```py
+   client = IdentityMapClient(base_url, api_key, client_secret)
+   ```
+2. Call a function that takes the timestamp string as input and generates an `IdentityBucketsResponse` object. The timestamp string should be in ISO 8601 format `YYYY-MM-DD[*HH[:MM[:SS[.fff[fff]]]][+HH:MM[:SS[.ffffff]]]]`.
+The below examples are valid timestamp strings.
+   1. Date in local timezone: `2024-08-18`
+   2. Date and time in UTC: `2024-08-18T14:30:15.123456+00:00`
+   3. Date and time in EST: `2024-08-18T14:30:15.123456-05:00`
+
+   ```py
+      since_timestamp = '2024-08-18T14:30:15+00:00'
+      identity_buckets_response = client.get_identity_buckets(datetime.fromisoformat(since_timestamp))
+   ```
+3. Iterated through the list of rotated salt buckets and extract the bucket_id and last_updated timestamp as follows
```

Comment:
Iterate? If it's a step, not sure why it would be iterated.
bucket_id and last_updated_timestamp both should have the code formatting (single ticks for inline).

Reply:
the response is a list of buckets and there is a`for` loop in the sample to iterate through the list

Reply:
thx